Include ocm barcoding handler in cellranger/multi module

fmalmeida opened the pull request:
Closes #8804

grst commented on modules/nf-core/cellranger/multi/templates/cellranger_multi.py:
```diff
@@ -154,6 +154,10 @@ def chunk_iter(seq, size):
         config_txt = config_txt + "\\n${include_frna}\\n" + input_conf.read() + "\\n"
 
+if len("${include_ocm}") > 0:
+        with open("${ocm_csv_text}") as input_conf:
+            config_txt = config_txt + "\\n${include_ocm}\\n" + input_conf.read() + "\\n"
```
nit: fix indent or run black/ruff

grst reviewed (commented):
new parameter added, should probably be reflected in meta.yaml?

grst commented:
I *think* they are mutually exclusive. for Frna and cmo I'm pretty sure they are.

fmalmeida commented:
Hi @grst ,

I have updated the `meta.yml`, fixed the linting, and added a checker.

Do you think the message in the checker suffices or should be different?

grst reviewed (approved):
LGTM
